code/datums/brain_damage/brain_trauma.dm

```diff
@@ -12,6 +12,9 @@
 	var/can_gain = TRUE //can this be gained through random traumas?
 	var/resilience = TRAUMA_RESILIENCE_BASIC //how hard is this to cure?
 
+/datum/brain_trauma/New()
+	return
```

Comment:
???

Reply:
oops i left it there while i was trying to fix a byond bug